### DISCUSSION TITLE
Request Logging

### DISCUSSION THREAD

**Discussion by jensenbox** [eadwinCode/django-ninja-extra#94] [Ideas]
What would I need to do if I wanted to implement something like what Stripe has where it logs the inbound and outbound requests against an account? I suspect this could tie in to the throttling mechanism some how but that might be too many things in the same basket.

I think a middleware or hook (or even signals) would be an interesting route to take but where would I wire it up?

**Comment by eadwinCode** [comment]
@jensenbox I don't really know who stripe works but if you want to wire up a middleware, I think Django has a good documentation on that and it applies to Django Ninja and NinjaExtra as well. 
You can also look at [structlog](https://github.com/jrobichaud/django-structlog) if that helps give you an insight on logging requests
